python/google/protobuf/internal/message_python_test.py

```diff
@@ -1,54 +0,0 @@
-#! /usr/bin/python
```

Comment:
I'm not sure, Python is not my first language. For now, we're definitely
testing both in Travis, though the setup is a bit brittle.

Let's move forward with this for now. Maybe open an inside an issue to
track your suggestion?
On Apr 10, 2015 4:20 PM, "Joshua Haberman" <notifications@github.com> wrote:

> In python/google/protobuf/internal/message_python_test.py
> <https://github.com/google/protobuf/pull/276#discussion_r28177045>:
>
> > @@ -1,54 +0,0 @@
> > -#! /usr/bin/python
>
> I agree that these tests are obsolete and should be removed.
>
> However I would like to (not necessarily in this PR) to add two tests that
> verify we are testing the pure-Python and the C++ implementation,
> respectively. But to do that we'd need a way to conditionally run either
> one test or the other, depending on whether --cpp_implementation was
> specified or not. Do you know an easy way to do that?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/protobuf/pull/276/files#r28177045>.
>